packages/appstore/src/components/containers/currency-switcher-container.tsx

```diff
@@ -2,15 +2,52 @@ import React, { HTMLAttributes, ReactNode } from 'react';
 import classNames from 'classnames';
 import { Icon } from '@deriv/components';
 import CurrencyIcon, { Currency } from 'Assets/svgs/currency';
-import './currency-switcher-container.scss';
 import { useStore, observer } from '@deriv/stores';
+import TradingPlatformIcon from 'Assets/svgs/trading-platform';
+import './currency-switcher-container.scss';
 
 interface CurrentSwitcherContainerProps extends Omit<HTMLAttributes<HTMLDivElement>, 'title'> {
     actions?: ReactNode;
     has_interaction?: boolean;
-    icon: Currency;
+    icon: Currency | 'Options';
     title: ReactNode;
+    show_dropdown?: boolean;
 }
+type CurrencyPlatformIconProps = {
+    icon: Currency | 'Options';
+};
+
+type DropdownProps = Omit<
+    CurrentSwitcherContainerProps,
+    'actions' | 'children' | 'className' | 'has_interaction' | 'icon' | 'title' | 'show_dropdown'
+>;
+
+const CurrencyPlatformIcon = ({ icon }: CurrencyPlatformIconProps) =>
+    icon === 'Options' ? (
+        <TradingPlatformIcon icon={icon} size={32} className='currency-switcher__currency--icon' />
+    ) : (
+        <CurrencyIcon icon={icon} size={32} className='currency-switcher__currency--icon' />
+    );
+
+const Dropdown = observer(({ ...props }: DropdownProps) => {
+    const store = useStore();
+    const { modules, traders_hub } = store;
+    const { is_eu_user, is_demo } = traders_hub;
+    const { current_list } = modules.cfd;
+
+    const has_mf_mt5_account = Object.keys(current_list)
+        .map(key => current_list[key])
+        .some(account => account.landing_company_short === 'maltainvest');
```

Comment:
This is checking mt5 jurisdiction . right? can we use Jurisdiction from @deriv/shared and use as below
`.some(account => account.landing_company_short === Jurisdiction.MALTA_INVEST); `

Reply:
sorted